problemsets-mat223/linearalbegra.tex

```diff
@@ -3861,31 +3863,134 @@ \section*{Task 2.3: Pat and Jamie}
 	\end{definition}
 
 	\question
 	Consider $A=\mat{1&0&0\\0&1&0}$.
 	\begin{parts}
 		\item Describe the row space of $A$.
+			\begin{solution}
+				$\Row(A) = \Span\Set*{\mat{1\\0\\0},\mat{0\\1\\0}}$,
+				which is the $xy$-plane in $\R^3$. 
+			\end{solution}
 		\item Describe the column space of $A$.
+			\begin{solution}
+				$\Col(A)=\Span\Set*{\mat{1\\0},\mat{0\\1},\mat{0\\0}}
+					=\Span\Set*{\mat{1\\0},\mat{0\\1}}=\R^2$.
+			\end{solution}
 		\item Is the row space of $A$ the same as the column space of $A$?
+			\begin{solution}
+				No. 
+
+				Although they are both two dimensional spaces, $\Row(A)$ is a
+				subspace of $\R^3$ and all vectors in it have three coordinates 
+				(with the third always being zero), while $\Col(A)$ is a
+				subspace of $\R^2$ and all vectors in it have two coordinates. 
+				Therefore, these two spaces are different.
+			\end{solution}
 		\item Describe the set of all vectors perpendicular to the rows of $A$.
+			\begin{solution}
+				The $z$-axis in $\R^3$. 
+
+				A vector $\vec x=\mat{x\\y\\z}$ is perpendicular to the rows of $A$
+				if and only if its dot product with both rows is zero. That is
+				\[
+					\vec x\cdot\mat{1\\0\\0} = x = 0
+					\quad \text{and} \quad
+					\vec x\cdot\mat{0\\1\\0} = y = 0.
+				\]
+				$\vec x$ satisfies these equations if and only if $\vec x=\mat{0\\0\\t}$
+				for some real number $t$, or in other words if $\vec x$ is on the
+				$z$-axis. 
+			\end{solution}
 		\item Describe the null space of $A$.
-		\item Describe the range and null space of $T_A$, the transformation induced
-			by $A$.
+			\begin{solution}
+				The $z$-axis in $\R^3$. 
+
+				A vector $\vec x=\mat{x\\y\\z}$ is in $\Null(A)$ if
+				and only if
+				\[
+					A\vec x = \mat{x\\y} = \vec 0.
+				\]
+				These are the same conditions as in the previous part, so the set
+				of vectors satisfying this is the $z$-axis.	
+			\end{solution}
+		\item Describe the range and null space of $T_A$, the transformation
+			induced by $A$.
+			\begin{solution}
+				$\Range(T_A)=\Set{\vec v\in\R^2 \given [\vec v]_{\mathcal E}\in\Col(A)}=\R^2$	
+				and $\Null(T_A)=\Set{\vec v\in\R^3 \given [\vec v]_{\mathcal E}\in\Null(A)}$, 
+				which is the $z$-axis in $\R^3$. 
+
+				First, by definition, $\vec v\in\Range(T_A)$ when 
+				$[\vec v]_{\mathcal E}=[T_A\vec x]_{\mathcal E}=A[\vec x]_{\mathcal E}$,
+				for some $\vec x\in\R^3$. But for any vector $\vec x\in\R^3$, if 
+				$[\vec x]_{\mathcal E}=\mat{x\\y\\z}$, we know that 
+				\[
+					A[\vec x]_{\mathcal E} = x\mat{1\\0}+y\mat{0\\1}.
+				\]
+				In other words $A[\vec x]_{\mathcal E}$ can be anything in $\Col(A)$,
+				and so $\Range(T_A)$ is the set of all vectors whose representation
+				in the standard basis is in $\Col(A)$. We know from a previous part
+				that $\Col(A)=\R^2$, and so any vector's representation in the
+				standard basis is in $\Col(A)$. 
```

Comment:
This is way too much of an argument. range(T_A) = span{T_A e_1, T_A e_2, T_A e_3} = span{ A[e_1], A[e_2], A[e_3] } = col(A).

The only thing that is a little annoying about this argument is the mixing of column vectors and vectors in R^n...but I don't think it's worth being 100% "correct" with the notation.

Reply:
Ah, okay. I can do that. 

I thought as I was writing that that it was very complicated for something that should be very simple, but being very careful with the vector/column vector distinction sort of led me there.

Reply:
Actually wait, I just noticed that in 46.3 we show that the range of an induced transformation equals the span of the columns of the matrix. Can I just reference that?

If not and I do it like you suggest, should I justify that range(T_A) = span{T_A e_1, T_A e_2, T_A e_3}?

Reply:
Yeah. You can reference a previous problem!